src/fracas/hover-provider.ts

```diff
@@ -12,13 +12,22 @@ export class FracasHoverProvider implements vscode.HoverProvider {
         return this._searchComment(document, position, token);
     }
 
-    private async _searchComment(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken)
-        : Promise<vscode.Hover | null> {
-        const [symbol, range] = getSelectedSymbolRange(document, new vscode.Range(position, position));
-        const definitions = await findDefinition(symbol, token);
+    private async _searchComment(
+        document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken
+    ): Promise<vscode.Hover | null> {
+        // search for comments preceding the definition for the symbol at the cursor
```

Comment:
another feature for later would be to extract the `#:tool-tip` struct field properties and add them here.

Reply:
What is the `#:tool-tip` property? I noticed it for the first time yesterday. Is it supported on all struct defs? I thought it was specific to some type def that was being used to feed a UMG widget or the Unreal details panel.